Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -48,6 +48,27 @@ $ldapBaseDn = $installSettings.ldapBaseDn
 
 $workingDirectory = Get-CurrentScriptDirectory
 
+try{
+	$signingCert = Get-Certificate $primarySigningCertificateThumbprint
+}catch{
+	Write-Host "Could not get signing certificte with thumbprint $primarySigningCertificateThumbprint. Please verify that the primarySigningCertificateThumbprint setting in install.config contains a valid thumbprint for a certificate in the Local Machine Personal store."
```

Comment:
Minor - certificate misspelled in error message.

Reply:
Thanks for catching that. I've updated it.